crates/indexer/src/models/move_resources.rs

```diff
@@ -23,6 +23,7 @@ pub struct MoveResource {
     pub generic_type_params: Option<serde_json::Value>,
     pub data: Option<serde_json::Value>,
     pub is_deleted: bool,
+    pub state_key_hash: String,
```

Comment:
What is this for again?

Reply:
This is the differentiator between resources in a resource group.

Reply:
yea we need this to know what's in a resource group. It's the identifier for slot I think.

Reply:
How is this used?

Reply:
Not sure rn. Just creating the data models. Probably will need to look up to know what items are deleted.